modules/Administration/language/en_us.lang.php

```diff
@@ -531,8 +531,10 @@
     'LBL_THEME_SETTINGS_DESC' => 'Choose themes for users to be able to select',
     'LBL_TIMEZONE' => 'Time Zone',
     'LBL_TO' => ' to ',
-    'LBL_SUITEFEED_SETTINGS' => 'Activity Streams',
-    'LBL_SUITEFEED_SETTINGS_DESC' => 'Enable the user feed and module feeds for the My Activity Stream dashlet',
+    'LBL_SUGARFEED_SETTINGS' => 'Activity Streams',
```

Comment:
this should be 
`'LBL_SUGARFEED_SETTINGS' => 'Activity Stream Settings',`
see https://github.com/salesagility/SuiteCRM/issues/5527